src/App.js

```diff
@@ -1,29 +1,40 @@
 import React from 'react';
+import './i18n';
 import '../src/assets/styles/main.scss';
-import Header from './components/Organisms/header/Header';
-import Home from './components/Pages/Home/Home';
 import Products from './components/Pages/Products/Products';
+import Header from './components/Organisms/header/Header';
 import Signin from './components/Pages/Signin/Signin';
+import Home from './components/Pages/Home/Home';
+import Footer from './components/Organisms/Footer/Footer';
 import Register from './components/Pages/Register/Register';
-import {Route, Switch, HashRouter as Router, Redirect} from 'react-router-dom';
 import {useTranslation} from 'react-i18next';
-import './i18n';
+import { Cart } from './components/Pages/Cart/Cart';
+import { GlobalProvider } from './Context/globalContext';
+import { Dialog } from './components/Pages/Modal/Dialog';
+import {Route, Switch, HashRouter as Router, Redirect} from 'react-router-dom';
 
 const App =()=>{
-    // For using locale in the application
-    const {t, i18n} = useTranslation();
+    // For using locale in the application. t is the core translation function, which through our key value passed
+    // as param fetches the related value from our languages objects file. 'i18n' holds many properties used 
+    // for getting current state of langugae in the application and updating it.
+    const {t} = useTranslation();
```

Comment:
where it's being used?

Reply:
It was mentioned in i18next docs, thus added it. I'll check by commenting it too.